### PR TITLE
Added Mariadb Connection and Method for Scanning MySQL Tables

### PR DESCRIPTION
**Pull request by ScottPiersall**

### CODE REVIEW

**Review by zachrussell12 — APPROVED** [review]

File: src/RoundTabler/PCIScan.java [hunk 1]
```diff
@@ -13,12 +16,66 @@
 	// with embedded dashes
 	static String CardNumberSequenceRegex = "\b(?:\\d[-]*?){13,16}\b";
 	static Pattern CardNumberPattern = Pattern.compile( CardNumberSequenceRegex);
+	
+	public String JDBC_DRIVER;
+
+	public string ScanResult;
```
Comment (theblasher) [inline src/RoundTabler/PCIScan.java:22]: this needs to be updated to "String" not "string"

File: src/RoundTabler/PCIScan.java [hunk 2]
```diff
@@ -13,12 +16,66 @@
...
 
+	private Conenction pDBConnection;
```
Comment (theblasher) [inline src/RoundTabler/PCIScan.java:24]: Connection is spelled incorrectly

File: src/RoundTabler/PCIScan.java [hunk 3]
```diff
@@ -13,12 +16,66 @@
...
+	private String pTableName; 
+	private String pMYSQLColumnSelect;
+	private String pMYSQLColumnTypes;
 
 	public PCIScan(){
 		super();
+		this.pMYSQLColumnTypes = "(mediumtext, longtext, text, tinytext, varchar)";
+	}
+
+	public PCIScan(Connection MYSqlDBConnection, string TableName  ){
```
Comment (theblasher) [inline src/RoundTabler/PCIScan.java:34]: Again, "String" needs to be capitalized

File: src/RoundTabler/PCIScan.java (hunk 2 above)
Comment (ScottPiersall) [reply src/RoundTabler/PCIScan.java:24]: Fixed

File: src/RoundTabler/PCIScan.java (hunk 3 above)
Comment (ScottPiersall) [reply src/RoundTabler/PCIScan.java:34]: Fixed

File: src/RoundTabler/PCIScan.java (hunk 1 above)
Comment (ScottPiersall) [reply src/RoundTabler/PCIScan.java:22]: Fixed

**Pull request merged**